<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Table-Wizard</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/app_icon.png</normaloff>../img/app_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="topMargin">
     <number>16</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item>
     <layout class="QGridLayout" name="central_grid_layout">
      <item row="2" column="0">
       <layout class="QGridLayout" name="bottom_grid_layout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="columns_selected_label">
          <property name="styleSheet">
           <string notr="true">color:gray</string>
          </property>
          <property name="text">
           <string>Columns selected:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QLabel" name="path_label">
          <property name="styleSheet">
           <string notr="true">color:gray</string>
          </property>
          <property name="text">
           <string>Path:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="rows_selected_label">
          <property name="styleSheet">
           <string notr="true">color:gray</string>
          </property>
          <property name="text">
           <string>Rows selected:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QFrame" name="table_frame">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QTableWidget" name="table_widget"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="table_label">
            <property name="styleSheet">
             <string notr="true">font-size:18px;</string>
            </property>
            <property name="text">
             <string>Table</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="tools_frame">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="editor_tab">
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <attribute name="title">
              <string>Editor</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="editor_label">
                <property name="styleSheet">
                 <string notr="true">font-size:18px;</string>
                </property>
                <property name="text">
                 <string>Editor</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="editor_tools_scroll_area">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="editor_tools_widget">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>258</width>
                   <height>473</height>
                  </rect>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="2" column="0">
                   <widget class="QLabel" name="value_label">
                    <property name="text">
                     <string>Value</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="selected_items_label">
                    <property name="text">
                     <string>Selected items</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="editor_tools_edit_label">
                    <property name="styleSheet">
                     <string notr="true">font-size:16px</string>
                    </property>
                    <property name="text">
                     <string>Edit table</string>
                    </property>
                   </widget>
                  </item>
                  <item row="8" column="0" colspan="2">
                   <widget class="QPushButton" name="insert_column_button">
                    <property name="text">
                     <string>Insert column</string>
                    </property>
                   </widget>
                  </item>
                  <item row="7" column="0" colspan="2">
                   <widget class="QPushButton" name="insert_row_button">
                    <property name="text">
                     <string>Insert row</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="0" colspan="2">
                   <widget class="QPushButton" name="add_row_button">
                    <property name="text">
                     <string>Add row</string>
                    </property>
                    <property name="checkable">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="value_edit">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="13" column="0" colspan="2">
                   <widget class="QPushButton" name="order_columns_button">
                    <property name="text">
                     <string>Order columns</string>
                    </property>
                   </widget>
                  </item>
                  <item row="9" column="0" colspan="2">
                   <widget class="QPushButton" name="delete_row_button">
                    <property name="text">
                     <string>Delete row</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="selected_items_edit">
                    <property name="styleSheet">
                     <string notr="true">color: gray</string>
                    </property>
                    <property name="text">
                     <string>None</string>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="0" colspan="2">
                   <widget class="QPushButton" name="add_column_button">
                    <property name="text">
                     <string>Add column</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" colspan="2">
                   <widget class="QLabel" name="label">
                    <property name="styleSheet">
                     <string notr="true">font-size:16px</string>
                    </property>
                    <property name="text">
                     <string>Edit selected items</string>
                    </property>
                   </widget>
                  </item>
                  <item row="11" column="0">
                   <widget class="QLabel" name="editor_tools_order_label">
                    <property name="styleSheet">
                     <string notr="true">font-size:16px;</string>
                    </property>
                    <property name="text">
                     <string>Order</string>
                    </property>
                   </widget>
                  </item>
                  <item row="12" column="0" colspan="2">
                   <widget class="QPushButton" name="order_rows_button">
                    <property name="text">
                     <string>Order rows</string>
                    </property>
                   </widget>
                  </item>
                  <item row="15" column="0">
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="10" column="0" colspan="2">
                   <widget class="QPushButton" name="delete_column_button">
                    <property name="text">
                     <string>Delete column</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0" colspan="2">
                   <widget class="QPushButton" name="edit_headers_button">
                    <property name="text">
                     <string>Edit headers</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="analysis_tab">
             <attribute name="title">
              <string>Analysis</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="analysis_label">
                <property name="styleSheet">
                 <string notr="true">font-size:18px;</string>
                </property>
                <property name="text">
                 <string>Analisys</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="analysis_tools_scroll_area">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>203</width>
                   <height>408</height>
                  </rect>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="3" column="1">
                   <widget class="QComboBox" name="filter_type_combo_box">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <item>
                     <property name="text">
                      <string>Value</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Function</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="4" column="0" colspan="2">
                   <widget class="QPushButton" name="select_button">
                    <property name="text">
                     <string>Select</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="0" colspan="2">
                   <widget class="QTextBrowser" name="finding_browser">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>196</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="selection_color_edit">
                    <property name="text">
                     <string>#007fff</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="selection_color_label">
                    <property name="text">
                     <string>Selection color:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="filter_type_label">
                    <property name="text">
                     <string>Filter type</string>
                    </property>
                   </widget>
                  </item>
                  <item row="11" column="0" colspan="3">
                   <spacer name="verticalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="0" column="0" colspan="3">
                   <widget class="QLabel" name="label_2">
                    <property name="styleSheet">
                     <string notr="true">font-size:16px;</string>
                    </property>
                    <property name="text">
                     <string>Selection</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="filter_edit">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="filter_label">
                    <property name="text">
                     <string>Filter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="10" column="0" colspan="2">
                   <widget class="QPushButton" name="clear_selections_button">
                    <property name="text">
                     <string>Clear selections</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_create"/>
    <addaction name="separator"/>
    <addaction name="action_import"/>
    <addaction name="action_open_table"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_export"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_add_row"/>
    <addaction name="action_add_column"/>
    <addaction name="action_insert_row"/>
    <addaction name="action_insert_column"/>
    <addaction name="action_delete_row"/>
    <addaction name="action_delete_column"/>
    <addaction name="separator"/>
    <addaction name="action_order_rows"/>
    <addaction name="action_order_columns"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>History</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_last"/>
    <addaction name="actionClear_all"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menuHistory"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_create">
   <property name="icon">
    <iconset>
     <normaloff>../img/create_icon.png</normaloff>../img/create_icon.png</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="action_import">
   <property name="icon">
    <iconset>
     <normaloff>../img/import_icon.png</normaloff>../img/import_icon.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset>
     <normaloff>../img/save_icon.png</normaloff>../img/save_icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_add_column">
   <property name="text">
    <string>Add column</string>
   </property>
  </action>
  <action name="action_add_row">
   <property name="text">
    <string>Add row</string>
   </property>
  </action>
  <action name="action_insert_column">
   <property name="text">
    <string>Insert column</string>
   </property>
  </action>
  <action name="action_insert_row">
   <property name="text">
    <string>Insert row</string>
   </property>
  </action>
  <action name="action_delete_column">
   <property name="text">
    <string>Delete column</string>
   </property>
  </action>
  <action name="action_delete_row">
   <property name="text">
    <string>Delete row</string>
   </property>
  </action>
  <action name="action_TXT">
   <property name="text">
    <string> TXT</string>
   </property>
  </action>
  <action name="actionCSV">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="action_order_rows">
   <property name="text">
    <string>Order rows</string>
   </property>
  </action>
  <action name="action_order_columns">
   <property name="text">
    <string>Order columns</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="action_last">
   <property name="text">
    <string>Select last action</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select action...</string>
   </property>
  </action>
  <action name="actionClear_all">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="action_open_table">
   <property name="icon">
    <iconset>
     <normaloff>../img/open_icon.png</normaloff>../img/open_icon.png</iconset>
   </property>
   <property name="text">
    <string>Open table</string>
   </property>
  </action>
  <action name="action_export">
   <property name="icon">
    <iconset>
     <normaloff>../img/export_icon.png</normaloff>../img/export_icon.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionAdd_finding">
   <property name="text">
    <string>Add Selection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
